app/src/main/res/xml/locales_config.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<locale-config xmlns:android="http://schemas.android.com/apk/res/android">
+    <locale android:name="en"/>
+    <locale android:name="ca"/>
+    <locale android:name="cs"/>
+    <locale android:name="cy"/>
+    <locale android:name="de"/>
+    <locale android:name="en-gb"/>
```

Comment:
is it intended that it's `en-gb` here but `en-GB` in `donottranslate.xml`?

Reply:
nope should be the same everywhere, thx